src/core/shared_state/currentEvent.ts

```diff
@@ -10,7 +10,7 @@ export type CurrentEventAtomState = {
 
 export const currentEventAtom = createAtom(
   {
-    setCurrentEventId: (eventId: string) => eventId,
+    setCurrentEventId: (eventId: string | null) => eventId,
```

Comment:
It is better to create the new action e.g. `deselectCurrentEvent`

Reply:
I'd not implement extra actions for such trivial things, because it will just add more "noise" for reatomv3 migration